ConcurrentModificationExceptionResolved/src/com/webonise/concurrentmodification/ConcurrentModification.java

```diff
@@ -0,0 +1,31 @@
+package com.webonise.concurrentmodification;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+/**
+ * Created by webonise on 19/9/16.
+ */
+public class ConcurrentModification {
+    public static void main(String args[]) {
+        //To avoid  ConcurrentModificationException we will useCopyOnWriteArrayList
+        List<String> myList = new CopyOnWriteArrayList<>();
+
+        myList.add("first");
+        myList.add("second");
+        myList.add("third");
+        myList.add("fourth");
+        myList.add("fifth");
+
+        Iterator<String> it = myList.iterator();
+        while (it.hasNext()) {
+            String value = it.next();
+            System.out.println("List:" + value);
+            if (value.equals("third")){
+                myList.remove(value);
```

Comment:
What is the point of using `Iterator` if directly `myList` is being used to modify list (`myList.remove(value)`)?